microsoft/botframework-solutions
# [Virtual Assistant Template][TypeScript] Add new Sample Skill

## Description
	
- Add Sample Skill

Related #1199 
	
## Testing Steps

1. Go to `.AI/templates/Virtual-Assistant-Template/typescript/samples/sample-skill`
1. Open a terminal in this location
1. Run the `deploy.ps1` (which is inside deployment\scripts folder) to deploy the sample skill
1. Verify that the deployment finished successfully
1. Run the command `npm install` and `npm run build`
1. Run the sample-skill with the command `npm run start`
1. Check that the sample has the same behavior as C# version
	
## Checklist
*-*
